src/rinterface_extra.c

```diff
@@ -2299,9 +2299,9 @@ igraph_error_t R_igraph_interrupt_handler(void *data) {
    */
   if (R_ToplevelExec(checkInterruptFn, NULL) == FALSE) {
     IGRAPH_FINALLY_FREE();
-    return IGRAPH_INTERRUPTED;
+    return 1;
```

Comment:
It's a good idea to use `true` / `false` instead of `1` and `0`. That way it's immediately clear that what's returned is a boolean, not an integer. The same goes for `NULL` vs `0`.